buffer/test/com/treode/buffer/PagedBufferSpec.scala

```diff
@@ -48,33 +48,52 @@ class PagedBufferSpec extends FlatSpec {
   capacity (64, 2)
   capacity (65, 3)
 
-  def discard (nbytes: Int, npages: Int) {
-    it should (s"discard the right pages for nbytes=$nbytes") in {
+  def discard (nbytes: Int, npages: Int, wbytes: Int, rbytes: Int, dbytes: Int) {
```

Comment:
I'm not clear what's happening here. I think you only need one new variable: discarded.